android/src/test/java/com/revenuecat/purchases/common/MappersTests.kt

```diff
@@ -2,27 +2,27 @@ package com.revenuecat.purchases.common
 
 import com.revenuecat.purchases.common.mappers.toMillis
 import org.assertj.core.api.Assertions.assertThat
-import org.spekframework.spek2.Spek
-import org.spekframework.spek2.style.specification.describe
+import org.junit.jupiter.api.DisplayName
+import org.junit.jupiter.api.Nested
+import org.junit.jupiter.api.Test
 import java.text.SimpleDateFormat
 import java.util.*
 
-object MappersHelpersTests : Spek({
+internal class MappersHelpersTests {
 
-    describe("when mapping dates to milliseconds") {
+    @Test
+    fun `PurchasesPeriod gets parsed correctly`() {
```

Comment:
this class doesn't really specify anything related to `PurchasesPeriod`, I think we should rename the tests accordingly - perhaps something like
```kotlin
fun `parsing simple date format returns the correct date`() {
```

Reply:
my brain was really fried